INSTALL.rst

```diff
@@ -0,0 +1,363 @@
+
+The Windows wheels (:file:`*.whl`) on the `PyPI download page
+<https://pypi.python.org/pypi/matplotlib/>`_ do not contain test data
+or example code.
+If you want to try the many demos that come in the Matplotlib source
+distribution, download the :file:`*.tar.gz` file and look in the
+:file:`examples` subdirectory.
+To run the test suite:
+
+ * extract the :file:`lib\\matplotlib\\tests` or
+   :file:`lib\\mpl_toolkits\\tests` directories from the source distribution;
+ * install test dependencies: `pytest <https://pypi.python.org/pypi/pytest>`_,
+   `mock <https://pypi.python.org/pypi/mock>`_, Pillow, MiKTeX, GhostScript,
+   ffmpeg, avconv, mencoder, ImageMagick, and `Inkscape
+   <https://inkscape.org/>`_;
+ * run ``py.test path\\to\\tests\\directory``.
+
+.. note::
+
+   The following backends work out of the box: Agg, TkAgg, ps, pdf and svg.
+   TkAgg is probably the best backend for interactive use from the standard
+   Python shell or from IPython and is enabled as default.
+
+   GTK3 is not supported on windows.
+
+   For support for other backend, LaTeX rendering, animation input/output and
+   larger selection of file format, you may need to install `additional
+   dependencies <_install_requirements>`.
+
+
+.. _install_from_source:
+
+Installing from source
+======================
+
+If you are interested in contributing to Matplotlib development,
+running the latest source code, or just like to build everything
+yourself, it is not difficult to build Matplotlib from source.  Grab
+the latest *tar.gz* release file from `the PyPI files page
+<https://pypi.python.org/pypi/matplotlib/>`_, or if you want to
+develop Matplotlib or just need the latest bugfixed version, grab the
+latest git version :ref:`install-from-git`.
+
+The standard environment variables `CC`, `CXX`, `PKG_CONFIG` are respected.
+This means you can set them if your toolchain is prefixed. This may be used for
+cross compiling.
+::
+
+  export CC=x86_64-pc-linux-gnu-gcc
+  export CXX=x86_64-pc-linux-gnu-g++
+  export PKG_CONFIG=x86_64-pc-linux-gnu-pkg-config
+
+Once you have satisfied the requirements detailed below (mainly
+python, numpy, libpng and freetype), you can build Matplotlib.
+::
+
+  cd matplotlib
+  python setup.py build
+  python setup.py install
+
+We provide a `setup.cfg
+<https://raw.githubusercontent.com/matplotlib/matplotlib/master/setup.cfg.template>`_
+file that goes with :file:`setup.py` which you can use to customize
+the build process. For example, which default backend to use, whether
+some of the optional libraries that Matplotlib ships with are
+installed, and so on.  This file will be particularly useful to those
+packaging Matplotlib.
+
+If you have installed prerequisites to nonstandard places and need to
+inform Matplotlib where they are, edit ``setupext.py`` and add the base
+dirs to the ``basedir`` dictionary entry for your ``sys.platform``.
+e.g., if the header to some required library is in
```

Comment:
I think header to is correct. I am not the one who wrote this section, so I am going to leave it this one unless an native-english speaker can confirm or correct?